src/streamer/doceStream.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Org, StreamingClient, StatusResult } from '@salesforce/core';
+import { AnyJson, JsonMap } from '@salesforce/ts-types';
+import { Duration } from '@salesforce/kit';
+
+export abstract class DOCeStreaming {
```

Comment:
Just a thought, but do we want to put this behind an interface.  I know there are talks about maybe migrating this in the future to a different implementation.